tests/unit/api/v1/test_status.py

```diff
@@ -45,9 +47,18 @@ class HealthApiTestCase(test.TestCase):
 
     def setUp(self):
         super(HealthApiTestCase, self).setUp()
-        self.health_config = {"health": {
-            "endpoint": "http://dummy.example.org/api/health"
-        }}
+        self.health_config = {
+            "health": {
+                "endpoint": "http://dummy.example.org/api/health"
+            }
+        }
+
+        self.old_USE_FAKE_DATA = fake_api_base.USE_FAKE_DATA
+        fake_api_base.USE_FAKE_DATA = False
```

Comment:
Fake API is tested in ApiTestCases. I've restored all the lines deleted from there.

We can rename them as soon as there would be additional cases.